dash/Dashboard.py

```diff
@@ -707,26 +710,23 @@ def create_gui(self, dash_pos):
         window = tk.Tk()
         window.configure(bg="white")
 
-        screen_width = window.winfo_screenwidth()
-        screen_height = window.winfo_screenheight()
+        x, y, w, h = self.primary_monitor.x, self.primary_monitor.y, self.primary_monitor.width, self.primary_monitor.height
```

Comment:
move this size determination logic to GSServer. Dashboard should not know anything about `--dash-pos` or any other command line parameters.